src/main/java/com/igot/cb/org/consumer/OrgFrameworkConsumer.java

```diff
@@ -1,261 +1,267 @@
-            logger.error("Unexpected error occurred while creating term", e);
-        }
-    }
-
-    private void publishFramework(String fwName, String orgId) {
-        try {
-            StringBuilder strUrl = new StringBuilder(configuration.getKnowledgeMS());
-            strUrl.append(configuration.getFrameworkPublish()).append("/").append(fwName);
-            Map<String, String> headers = new HashMap<>();
-            headers.put(Constants.X_CHANNEL_ID, orgId);
-            Map<String, Object> response = outboundRequestHandlerServiceImpl.fetchResultUsingPost(
-                strUrl.toString(), "", headers);
-            if (response != null
-                && Constants.OK.equalsIgnoreCase((String) response.get(Constants.RESPONSE_CODE))) {
-                logger.info("Published the framework: {}", fwName);
-            } else {
-                logger.info("Unable to publish the framework with name: {}", fwName);
-                updateStatusToFailed(orgId);
-            }
-        } catch (Exception e) {
-            logger.error("Unexpected error occurred while publishing the framework", e);
-        }
-    }
-
-
-    public static Map<String, Object> createFrameworkRequest(String channelId, String frameworkName) {
-        Map<String, Object> framework = createFramework(channelId, frameworkName);
-        Map<String, Object> request = new HashMap<>();
-        request.put("framework", framework);
-        return request;
-    }
-
-    private static Map<String, Object> createFramework(String channelId, String frameworkName) {
-        Map<String, Object> framework = new HashMap<>();
-        StringBuilder name = new StringBuilder(channelId).append("_").append(frameworkName).append(Constants.MASTER);
-        framework.put(Constants.NAME, name);
-        framework.put(Constants.DESCRIPTION, "Master Framework Copy");
-        framework.put(Constants.CODE, name);
-        framework.put(Constants.OWNER, channelId);
-        return framework;
-    }
-
-    public static Map<String, Object> createRequestMap(Map<String, Object> termMap) {
-        Map<String, Object> requestMap = new HashMap<>();
-        requestMap.put("term", termMap);
-        return requestMap;
-    }
-
-    public static Map<String, Object> createOuterMap(Map<String, Object> requestMap) {
-        Map<String, Object> outerMap = new HashMap<>();
-        outerMap.put("request", requestMap);
-        return outerMap;
-    }
-
-    public static Map<String, Object> createTermMap(String termName, String category) {
-        Map<String, Object> termMap = new HashMap<>();
-        termMap.put(Constants.NAME, termName);
-        termMap.put(Constants.DESCRIPTION, termName);
-        termMap.put(Constants.CODE, UUID.randomUUID());
-        termMap.put(Constants.REF_TYPE, "");
-        termMap.put(Constants.REF_ID, "");
-        termMap.put(Constants.CATEGORY, category);
-        return termMap;
-    }
-
-
-}
+//package com.igot.cb.org.consumer;
```

Comment:
Just leave it as it is... or remove the class if not required... 
do not comment out entire file